### PR TITLE
pin dask to version 2.9.0, remove pinned environment files

### PR DESCRIPTION
**Pull request by slejdops**
Ref. #77

### CODE REVIEW

**Comment by slejdops** [comment]
pinning dask to 2.9.0 seems to have fixed the issue

**Comment by alimanfoo** [comment]
Thanks Krzysztof. I notice dask and distributed have both just released 2.9.2, can you see if they also fix the issue you were seeing with datalab?

**Comment by slejdops** [comment]
I have just tested an image (malariagen/datalab:1601test) with dask 2.9.2
I'mm getting the same errors as in 2.9.1

File: environment-pinned-linux.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: malariagen-3053b15
+me: malariagen-21ca02e
```
Comment (alimanfoo) [inline environment-pinned-linux.yml]:
```suggestion
name: malariagen-21ca02e
```

**Pull request merged**